src/transformers/models/clap/modeling_clap.py

```diff
@@ -0,0 +1,2479 @@
+        clipwise_output (`torch.FloatTensor` of shape `(batch_size, hidden_size)`):
+            Sequence of hidden-states at the output of the last layer of the model.
+        fine_grained_embedding (`torch.FloatTensor` of shape `(batch_size, num_frames, hidden_size)`):
+            Sequence of hidden-states at the output of the last layer of the model.
+        embedding (`torch.FloatTensor` of shape `(batch_size, hidden_size)`):
+            Sequence of hidden-states at the output of the last layer of the model.
+        attentions (`tuple(torch.FloatTensor)`, *optional*, returned when `output_attentions=True` is passed or when `config.output_attentions=True`):
+            Tuple of `torch.FloatTensor` (one for each layer) of shape `(batch_size, num_heads, sequence_length,
+        hidden_states (`tuple(torch.FloatTensor)`, *optional*, returned when `output_hidden_states=True` is passed or when `config.output_hidden_states=True`):
+            Tuple of `torch.FloatTensor` (one for the output of the embeddings, if the model has an embedding layer, +
+    """
+
+    framewise_output: torch.FloatTensor = None
+    clipwise_output: torch.FloatTensor = None
+    fine_grained_embedding: torch.FloatTensor = None
+    embedding: torch.FloatTensor = None
+    attentions: Optional[Tuple[torch.FloatTensor]] = None
+    hidden_states: Optional[Tuple[torch.FloatTensor]] = None
+
+
+@dataclass
+class CLAPAudioModelOutputWithProjection(ModelOutput):
+    """
+    CLAPAudio model output to mimic the output of the original implementation.
+
+    Args:
+        audio_embeds (`torch.FloatTensor` of shape `(batch_size, output_dim)`):
+            The audio embeddings obtained by applying the projection layer to the pooler_output.
+        framewise_output (`torch.FloatTensor` of shape `(batch_size, num_frames, hidden_size)`):
+            Sequence of hidden-states at the output of the last layer of the model.
+        clipwise_output (`torch.FloatTensor` of shape `(batch_size, hidden_size)`):
+            Sequence of hidden-states at the output of the last layer of the model.
+        fine_grained_embedding (`torch.FloatTensor` of shape `(batch_size, num_frames, hidden_size)`):
+            Sequence of hidden-states at the output of the last layer of the model.
+        embedding (`torch.FloatTensor` of shape `(batch_size, hidden_size)`):
+            Sequence of hidden-states at the output of the last layer of the model.
+        attentions (`tuple(torch.FloatTensor)`, *optional*, returned when `output_attentions=True` is passed or when `config.output_attentions=True`):
+            Tuple of `torch.FloatTensor` (one for each layer) of shape `(batch_size, num_heads, sequence_length,
+        hidden_states (`tuple(torch.FloatTensor)`, *optional*, returned when `output_hidden_states=True` is passed or when `config.output_hidden_states=True`):
+            Tuple of `torch.FloatTensor` (one for the output of the embeddings, if the model has an embedding layer, +
```

Comment:
Same as above.

Reply:
Fixed in [948eb0a](https://github.com/huggingface/transformers/pull/21370/commits/948eb0ae089a96605bded8ebcee9cb41131d63b5)